### PR TITLE
Update recipe for v1.0

### PR DESCRIPTION
**Pull request by adrn**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by adrn** [comment]
@conda-forge-admin, please rerender

**Comment by adrn** [comment]
@bsipocz @mwcraig or @beckermr - do you mind taking a look at this to make sure it seems correct? Thanks!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -27,7 +27,7 @@ requirements:
   run:
     - python <3.7
     - astropy
-    - six
+    - gsl
```
Comment (beckermr) [inline recipe/meta.yaml:31]: gsl in run but not host is really weird. is the library linking to gsl? Why would it only be required when running?

**Review by beckermr — CHANGES_REQUESTED** [review]
It appears the code requires python 3.5 or higher. Also, you probably need gsl in the host section too.

File: recipe/meta.yaml (hunk 1 above)
Comment (adrn) [reply recipe/meta.yaml:31]: Ah, oops!

**Comment by adrn** [comment]
Do I need to set a minimum python version somewhere?

**Comment by mwcraig** [comment]
@adrn -- one way to restrict the python version is here: http://conda-forge.org/docs/maintainer/adding_pkgs.html#build

**Comment by adrn** [comment]
@conda-forge-admin, please rerender

**Comment by beckermr** [comment]
@conda-forge-admin, please rerender

**Pull request merged**